### ISSUE TITLE
Is it possible add a transformation without any associated attribute?

### ISSUE THREAD

**Issue by andrewaguiar** [jamesmartin/inline_svg#36]
Is it possible add a transformation without any associated attribute? For instance, I would like to add a transformation to every svg without a specific attribute eg:

```ruby
InlineSvg.configure do |config|
  # Is it possible
  config.add_custom_transformation(transform: SetFocusableToFalseTransform)
end
```

and the transformation

```ruby
class SetFocusableToFalseTransform < InlineSvg::CustomTransformation
  def transform(doc)
    svg = doc.at_css('svg')
    svg['focusable'] = 'false'
    doc
  end
end
```

Thanks

**Comment by jamesmartin** [comment]
> Is it possible add a transformation without any associated attribute?

@andrewaguiar No, sorry. It's not possible, but I'm open to changing the gem to accommodate "static" custom transformations.

As a workaround I suggest rewriting your transformation like this:

```ruby
class SetFocusableTransform < InlineSvg::CustomTransformation
  def transform(doc)
    svg = doc.at_css('svg')
    svg['focusable'] = !!value.to_s
    doc
  end
end
```

And use it like this:

```ruby
inline_svg('some-document.svg', focusable: true)
inline_svg('some-document.svg', focusable: false)
```

**Comment by andrewaguiar** [comment]
@jamesmartin I sent a PR with an implementation of somthing that can be used to do this https://github.com/jamesmartin/inline_svg/pull/37

Thanks

**Comment by jamesmartin** [comment]
Fixed in [v0.8.0](https://github.com/jamesmartin/inline_svg/releases/tag/v0.8.0).
